File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-api/src/main/java/com/cbi/ccr/csw/dashboard/globalproperties/dto/GlobalPropertiesDTO.java
package com.cbi.ccr.csw.dashboard.globalproperties.dto;

import com.cbi.frw.api.dto.GenericDTO;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class GlobalPropertiesDTO extends GenericDTO {

	private String id;
	private String propertyName;
	private String value;
	private String type;
	private Boolean mandatory;
	private String requiredMsg;
	private Integer minLength;
	private Integer maxLength;
	private String displayName;
}

File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-ui/src/app/modules/configuration/global-properties/dto/global-properties-dto.ts
import { CommonConfigDTO, CommonConfigFilterDTO } from "../../common-config-dto";

export class GlobalPropertiesFilterDTO extends CommonConfigFilterDTO{

    propertyName!: string;
  
}

export class GlobalPropertiesDTO extends CommonConfigDTO{

    propertyName!: string;
    value!: string;
    type!: string;
    mandatory!: boolean;
    requiredMsg!: string;
    minLength!: number;
    maxLength!: number;
    displayName!: string;

 
}